pipeline/ffmpeg.go

```diff
@@ -31,15 +37,43 @@ func (f *ffmpeg) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 	job.SegmentingTargetURL = segmentingTargetURL.String()
 	log.AddContext(job.RequestID, "segmented_url", job.SegmentingTargetURL)
 
-	// TODO: Segment
+	// Begin Segmenting
 	log.Log(job.RequestID, "Beginning segmenting via FFMPEG/Livepeer pipeline")
 	job.ReportProgress(clients.TranscodeStatusPreparing, 0.5)
 
-	// TODO: Segmenting Finished
+	// FFMPEG fails when presented with a raw IP + Path type URL, so we prepend "http://" to it
+	internalAddress := config.HTTPInternalAddress
+	if !strings.HasPrefix(internalAddress, "http") {
+		internalAddress = "http://" + internalAddress
+	}
+
```

Comment:
Just FYI: Based on my other PR to handle HLS inputs in catalyst-api, I'm going to branch off here and skip segmenting if the input is HLS format.